docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+Openstaande vraag is backwards compatibility: lukt het om de ZDS API zo te ontwikkelen dat deze zowel binnen een GEMMA 2 landschap (huidige systemen/leveranciers) als een Gegevenslandschap gebruikt kan worden. Ook daarom is er eerst meer inzicht nodig in hoe (de informatiearchitectuur voor) ZGW in een Gegevenslandschap vorm krijgt.
+
+## 1.2 Veranderingen agv het Gegevenslandschap
+
+In GEMMA 2 worden processen zaakgericht uitgevoerd, ondersteund door informatiesystemen die invulling geven aan de generieke of specifieke Zaakafhandelcomponenten. De primaire verandering agv het gegevenslandschap zit in deze ondersteunende componenten.
+
+We signaleren drie grote veranderingen daarin: scheiding van gegevens en proceslogica, (eenmalige) opslag en (meervoudige) bevraging van gegevens bij (of door) de bron en (mede daardoor) zeer beperkte informatie-uitwisseling (geen gegevensdistributie of ‘rondpompen’, alleen nog notificaties) tussen procesapplicaties. We gaan hierop hieronder in.
+
+### 1.2.1. Opknippen Procesapplicaties
+
+In GEMMA 2 en bijgevolg in ZGW in GEMMA 2 zijn de taakspecifieke informatiesystemen “geherwaardeerd” en deze als “Specifieke Zaakafhandelcomponenten” neergezet. Processen worden afgehandeld met een Zaakafhandelcomponent (ZAC, generiek of specifiek). De generieke ZAC slaat haar gegevens op in de Zaakregistratiecomponent (ZRC). Daarentegen slaat de specifieke ZAC alle gegevens zelf op: over de zaak en het proces, het verloop daarvan en veelal ook de registratie van de objecten waarop dat proces betrekking heeft. Vanuit deze component wordt een zaak geregistreerd (gekopieerd) in de ZRC en wordt deze zaak bij wijzigingen (ook) daarin bijgewerkt. Volgens het Gegevenslandschap moeten deze gegevens gescheiden worden van de procesapplicatie en in een of meerdere registraties worden ondergebracht en via gestandaardiseerde API’s ontsloten.
+Voor de Generieke Zaakafhandelcomponent is hierin al voorzien: deze beperkt zich tot de gegevensset zoals wordt bijgehouden in de Zaakregistratiecomponent (ZRC). Voor de specifieke ZAC heeft dit (vergaande) consequenties. Merk op dat zaakgegevens nu potentieel dubbel worden vastgelegd: zowel in de specifieke ZAC als in de ZRC.
+
+### 1.2.2. Eenmalige opslag vanuit de bron
+
+Common Ground gaat uit van het beschikbaar stellen en bevragen van gegevens bij de bron. Kopiëren en distribueren van gegevens (gegevensmagazijnen, gegevensdistributie,..) bestaan in een gegevenslandschap niet meer.  ‘Beschikbaar stellen bij de bron’ moet evenwel niet altijd letterlijk genomen worden. Anders zouden er precies evenveel registraties als procesapplicaties ontstaan, wat een nogal gefragmenteerde gemeentelijke informatiehuishouding zou opleveren en waarbij bijvoorbeeld telkens alle bronregistraties bevraagd zouden moeten worden om een overzicht te krijgen van alle lopende zaken.
```

Comment:
Ik begrijp de conclusie niet dat "Beschikbaar stellen bij de bron" zou resulteren in precies evenveel registraties als procesapplicaties.

Reply:
verduidelijkt: elke procesapplicatie als zelfstandige bron voor de gegevens ontstaan in die applicatie